roles/reproducer/tasks/nat64_appliance.yml

```diff
@@ -0,0 +1,32 @@
+---
+# TODO(hjensas): Building takes a bit of time (~5 minutes)
+# We could host the image somewhere and download it instead.
+- name: Build nat64 appliance image
+  vars:
+    cifmw_nat64_appliance_run_dib_as_root: true
+  ansible.builtin.include_role:
+    name: nat64_appliance
+
+- name: Fix permissions on nat64_appliance dir - because we ran dib as root
+  become: true
+  ansible.builtin.file:
+    path: "{{ cifmw_basedir }}/nat64_appliance"
+    state: directory
+    recurse: true
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
+
+- name: Crate SSH keypair for nat64-appliance
+  register: _nat64_key
+  community.crypto.openssh_keypair:
+    comment: "test-key"
```

Comment:
care to add a better comment? "nat64-access-key" or something?

Reply:
Done.